### ISSUE TITLE
Disable built in ebook font

### ISSUE THREAD

**Issue by msmolka** [blchinezu/pocketbook-coolreader#229]
Hello
I have strange issue when disabled built in font (but internal styles enabled). For some reason cool reader is suing default pocket book font (dejavu sans I think) . I've set in every possible place (default font, fallback font, each style font) And still main text is in default font instead of set ones. If I disable internal styles then my font is applied properly. Otherwise I'm not able to use my font. I tried different default fonts without luck. Still default pocket book font.

**Comment by spcmd** [comment]
Yes, I also experienced this issue with Pocketbook 515 mini (pro4 -> FW4)

**Comment by msmolka** [comment]
I'm using Touch Lux 3 with latest firmware
